sdk/scenario-format/tests/interpreter_test.rs

```diff
@@ -31,6 +31,25 @@ fn test_string() {
     assert_eq!(EMPTY, interpret_string("str:", &context));
 }
 
+#[test]
+fn test_mxsc_pack() {
+    let context = &InterpreterContext::default();
+    assert_eq!(
+        b"test".to_vec(),
+        interpret_string("mxsc:tests/test.json", context)
+    );
+}
+
+#[should_panic]
```

Comment:
Unintuitively, should not panic.
When running using the debugger, we don't actually rely on these files. Their path is their sole identifier in this case, and only used as an identifier.
`file:` does not panic, neither should this. This fix should solve the failing rust tests in the CI.